corona-warn-app/cwa-app-android
# Censor private information in log files (EXPOSUREAPP-6516)

### Description
This PR adds censoring of the following private information for our error logging: 
- PCR QR-Code (GUID, URL) 
- PCR registration token 
- Created Event Information 
- Scanned Event Information 
- Rapid Test profile information 
- Rapid Test QR-Code (URL and JSON payload (encoded and decoded))
- Rapid test registration token 
- Personal Information in Rapid test result if exist

### How to test
- Start Error Logging
- Add PCR test, add RAT test, create Event, check in to an event, create a RAT profile
- Export Error Log to your computer
- Analyze the log and check if no private information is in the log
